sdks/python/apache_beam/runners/direct/evaluation_context.py

```diff
@@ -81,6 +81,7 @@ def __init__(self, side_inputs):
       self._transform_to_side_inputs[side.pvalue.producer].append(side)
 
   def __repr__(self):
+    # pylint: disable=dict-values-not-iterating
```

Comment:
Would s/`if self._views.values()`/`if self._views`/ remove the lint warning?

Reply:
You're right. I've removed the `values()` call.